### PR TITLE
feat: Parse services (RET-392)

### PR DESCRIPTION
**Pull request by DerekMaggio**
# Overview

Parse individual containers in Services for Docker Compose file.
Covers [RET-392](https://opentrons.atlassian.net/browse/RET-392)

# Changelog

- Add internal methods to parse services
- Add tests

# Review requests

None

# Risk assessment

Low

### CODE REVIEW

File: emulation_system/tests/compose_file_creator/test_conversion_logic.py [hunk 1]
```diff
@@ -2,26 +2,147 @@
...
+    TemperatureModuleImages,
+    ThermocyclerModuleImages,
+)
+from emulation_system.consts import DOCKERFILE_DIR_LOCATION
+
+ROBOT_NAME = "brobot"
+THERMOCYCLER_NAME = "temperamental"
+HEATER_SHAKER_NAME = "shakey-and-warm"
+TEMPERATURE_MODULE_NAME = "t00-hot-to-handle"
+MAGNETIC_MODULE_NAME = "fatal-attraction"
```
Comment (amitlissack) [inline emulation_system/tests/compose_file_creator/test_conversion_logic.py:43]: I love silly test names

**Review by amitlissack — APPROVED** [review]

File: emulation_system/tests/compose_file_creator/test_conversion_logic.py (hunk 1 above)
Comment (DerekMaggio) [reply emulation_system/tests/compose_file_creator/test_conversion_logic.py:43]: :)

Comment (Laura-Danielle) [reply emulation_system/tests/compose_file_creator/test_conversion_logic.py:43]: This gave me a chuckle 😆

**Review by Laura-Danielle — APPROVED** [review]
:)

**Pull request merged**